RadixWallet/Core/DesignSystem/Extensions/View+Extra.swift

```diff
@@ -45,4 +45,17 @@ extension View {
 			}
 		}
 	}
+
+	/// Sets the List section spacing if possible.
+	@available(iOS, deprecated: 18.0, message: "Should use native `listSectionSpacing` once iOS 16 is no longer supported.")
+	func setListSectionSpacing(_ spacing: CGFloat) -> some SwiftUI.View {
```

Comment:
`setX` should be reserved for setters, this could be called `withListSectionSpacing`